system_tests/resources/host-pool-service-blueprint/openstack-host-pool-service-blueprint.yaml

```diff
@@ -0,0 +1,73 @@
+##################################################################
+# this blueprint is used to install the cloudify-host-pool-service
+# on a host in an openstack manager environment
+##################################################################
+
+tosca_definitions_version: cloudify_dsl_1_0
+
+imports:
+  - http://www.getcloudify.org/spec/cloudify/3.2rc1/types.yaml
+  - http://www.getcloudify.org/spec/openstack-plugin/1.2rc1/plugin.yaml
+  - https://raw.githubusercontent.com/cloudify-cosmo/cloudify-host-pool-service/master/host-pool-service.yaml
+
+
+inputs:
+
+  image:
+    description: >
+      Image to be used when launching agent VM's
+
+  flavor:
+    description: >
+      Flavor of the agent VM's
+
+node_templates:
+
+  host_pool_service_host:
+    type: cloudify.openstack.nodes.Server
+    properties:
+      server:
+        image: { get_input: image }
+        flavor: { get_input: flavor }
+    relationships:
+      - target: host_pool_service_security_group
+        type: cloudify.openstack.server_connected_to_security_group
+      - target: host_pool_service_floating_ip
```

Comment:
remove floating ip. the service will be installed on the same network the management machine is on.